python/cudf/cudf/_lib/transform.pyx

```diff
@@ -156,3 +162,30 @@ def one_hot_encode(Column input_column, Column categories):
     )
 
     return encodings
+
+
+def compute_column(list columns, tuple column_names, expr: str):
+    """Compute a new column by evaluating an expression on a set of columns.
+
+    Parameters
```

Comment:
This docstring seems to mention `df` and `expr` where the function arguments appear to be `columns`, `column_names` and `expr`.